mockgcp/mockprivilegedaccessmanager/entitlement.go

```diff
@@ -121,6 +143,33 @@ func (s *PrivilegedAccessManager) DeleteEntitlement(ctx context.Context, req *pb
 	if err := s.storage.Delete(ctx, fqn, oldObj); err != nil {
 		return nil, err
 	}
+	metadata := constructOperationMetadata(fqn, "delete")
+	return s.operations.StartLRO(ctx, name.parent(), metadata, func() (proto.Message, error) {
+		result := proto.Clone(oldObj).(*pb.Entitlement)
+		result.State = pb.Entitlement_DELETED
+		result.Name = "projects/${projectNumber}/locations/global/entitlements/privilegedaccessmanagerentitlement-${uniqueId}"
+		now := timestamppb.New(time.Now())
+		metadata.EndTime = now
+		return result, nil
+	})
+}
+
+func computeEtag(obj proto.Message) string {
```

Comment:
nit: suggest using mockgcp/fields/etag.go

Reply:
Cool, will fix it in https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/2872